Swift Shift/src/mouse/MouseTracker.swift

```diff
@@ -72,53 +71,67 @@ class MouseTracker {
         print("Ignoring", bundleIdentifier)
         return true
     }
-    
+
     private func determineQuadrant(mouseLocation: NSPoint, windowSize: CGSize, windowLocation: NSPoint) -> Quadrant {
         let bounds = WindowManager.getWindowBounds(windowLocation: windowLocation, windowSize: windowSize)
-        
-        let midX = (bounds.topLeft.x + bounds.topRight.x) / 2
-        let midY = (bounds.topLeft.y + bounds.bottomLeft.y) / 2
-        
-        let isLeft = mouseLocation.x < midX
-        let isTop = mouseLocation.y > midY
-        
-        switch (isLeft, isTop) {
-        case (true, true):
+
+        let thirdX = (bounds.topRight.x - bounds.topLeft.x) / 3
+        let thirdY = (bounds.topLeft.y - bounds.bottomLeft.y) / 3
```

Comment:
I think this could allow for configurable centre size
```suggestion
        let centreSize = 0.24 // TODO: make this configurable
        let sideSize = (1 - centreSize) / 2
        let thirdX = (bounds.topRight.x - bounds.topLeft.x) * sideSize
        let thirdY = (bounds.topLeft.y - bounds.bottomLeft.y) * sideSize
```